lwreg/test_lwreg.py

```diff
@@ -356,6 +358,20 @@ def testStandardizationLibCheckers(self):
 ''',
                            config=lconfig), 2)
 
+    def testTimestamp(self):
+        cn = sqlite3.connect(':memory:')
+        self._config = utils.defaultConfig()
+        self._config['connection'] = cn
+        utils.initdb(config=self._config, confirm=True)
+        utils.register(smiles='CCC', config=self._config)
+        time.sleep(2)
+        utils.register(smiles='CCCC', config=self._config)
+        curs = cn.cursor()
+        d = curs.execute("select molregno, timestamp from orig_data")
```

Comment:
I believe that the order of results from a select are not guaranteed unless you provide an `order by` clause.

You can fix that here by doing:
 ```suggestion
        d = curs.execute("select molregno, timestamp from orig_data order by molregno asc")
```